pkg/chaosctl/recover/recover.go

```diff
@@ -40,12 +45,57 @@ type Recover interface {
 
```

Comment:
Reference to (effective_go)[https://go.dev/doc/effective_go#interface-names] , Recover can be named by Recoverer.
And I remember that STRRL suggest me to use name like Recoverable. Anyway , I think Recover is not a good interface name.

Reply:
Sure, I did not realize it😂

Reply:
Renamed